CMakePresets.json

```
@@ -12,9 +12,11 @@
           "TTNN_BUILD_TESTS": {"value": "TRUE"},
           "TT_UMD_BUILD_TESTS": {"value": "FALSE"},
           "BUILD_PROGRAMMING_EXAMPLES": {"value": "TRUE"},
-          "BUILD_TT_TRAIN": {"value": "TRUE"},
+          "BUILD_TT_TRAIN": {"value": "FALSE"},
           "ENABLE_CCACHE": {"value": "TRUE"},
-          "TT_UNITY_BUILDS": {"value": "FALSE"}
+          "TT_UNITY_BUILDS": {"value": "FALSE"},
+          "CMAKE_CXX_STANDARD": "20",
+          "CMAKE_CXX_STANDARD_REQUIRED": "ON"
```

```suggestion
          "TT_UNITY_BUILDS": {"value": "FALSE"}
```

Curious what you tripped over to need/want to add that.  It's right in the base project; no point repeating ourselves here in the preset.

---

basically just randomly throwing spaghetti at the wall in the hope that it would fix the span issue. It did feel redundant but I figured maybe it might change something.